packages/sdc/components/03-organisms/alert/alert.css

```
@@ -5,17 +5,20 @@
+  font-family: var(--ct-typography-text-small-font-name);
+  font-weight: var(--ct-typography-text-small-font-weight);
+  letter-spacing: var(--ct-typography-text-small-letter-spacing);
 }
 @media (min-width: 768px) {
   .ct-alert {
     padding: 1.5rem;
   }
 }
+.ct-alert .container {
+  position: relative;
+}
```

Stylelint (17.14.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)